src/lib/feed/index.js

```diff
@@ -16,21 +16,15 @@ import "rc-menu/assets/index.css";
 
 const AddButtonContainer = styled.div`
   display: flex;
-  width: 100%;
-  justify-content: flex-end;
-`;
+  justify-content: flex-start;
+  margin-bottom: ${props => props.theme.spacing.padding};
 
-// const AddButton = styled.a`
-//   cursor: pointer;
-//   color: ${props => props.theme.colors.darkPrimary};
-//   padding: ${props => props.theme.spacing.padding};
-//   user-select: none;
-//
-//   &:hover {
-//     color: ${props => props.theme.colors.primary};
-//     text-decoration: underline;
-//   }
-// `;
+  .rc-menu {
```

Comment:
With a white background, I'm not sure it even really looks like a clickable button. sort of...

Reply:
it does trust me brah